lib/livebook/hubs/team_client.ex

```diff
@@ -298,12 +314,29 @@ defmodule Livebook.Hubs.TeamClient do
       hub_id: state.hub.id,
       secrets: secrets,
       agent_keys: agent_keys,
+      app_deployments: app_deployments,
       clustering: nullify(deployment_group.clustering),
       zta_provider: atomize(deployment_group.zta_provider),
       zta_key: nullify(deployment_group.zta_key)
     }
   end
 
+  defp build_app_deployment(app_deployment) do
+    path = URI.parse(app_deployment.archive_url).path
+
+    %Teams.AppDeployment{
+      id: app_deployment.id,
+      filename: Path.basename(path),
+      slug: app_deployment.slug,
+      sha: app_deployment.sha,
+      title: app_deployment.title,
+      deployment_group_id: app_deployment.deployment_group_id,
+      file: app_deployment.archive_url,
```

Comment:
We put URL here, in other place we put the content. We should be more explicit, one way would be `archive: {:url, String.t()} | {:content, binary()} | nil`.